pkg/hostpath/hostpath.go

```diff
@@ -202,6 +204,30 @@ func getVolumePath(volID string) string {
 	return filepath.Join(dataRoot, volID)
 }
 
+func commonUpsertRanger(path string, key, value interface{}) bool {
+	buf, err := json.MarshalIndent(value, "", "    ")
+	if err != nil {
+		glog.Errorf("error marshalling: %s", err.Error())
+		return true
+	}
+	filePath := filepath.Join(path, fmt.Sprintf("%s", key))
+	glog.V(0).Infof("GGM create/update file %s", filePath)
```

Comment:
side note - if we depend on k8s, we should switch to klog.

Reply:
yeah for whatever reason https://github.com/kubernetes-csi/csi-driver-host-path uses `glog`

I'll handle the switch in a follow up PR right after this one merges